CHANGELOG.md

```diff
@@ -1,4 +1,24 @@
 ## 3.25.0 (Unreleased)
+FEATURES:
+* **New Resource:** `data_catalog_tag_template` ([#6485](https://github.com/terraform-providers/terraform-provider-google/pull/6485))
+* **New Resource:** `google_container_analysis_occurence` ([#6474](https://github.com/terraform-providers/terraform-provider-google/pull/6474))
+
+IMPROVEMENTS:
+* appengine: added `inbound_services` to `StandardAppVersion` resource ([#6514](https://github.com/terraform-providers/terraform-provider-google/pull/6514))
+* bigquery: Added `google_bigquery_table` `range_partitioning` to GA ([#6488](https://github.com/terraform-providers/terraform-provider-google/pull/6488))
```

Comment:
nit: `Added` here and `Promoted` for the GKE change below, may be worth changing for consistency 🤷‍♀️  (not worth it if you're not already editing the log)